src/config.rs

```diff
@@ -68,6 +73,72 @@ impl Config {
     }
 }
 
+#[derive(Serialize, Deserialize)]
+pub struct SubProcess {
+    pub command: String,
+    pub working_dir: Option<String>,
+    #[serde(default)]
+    pub keep_on_exit: bool,
+    #[serde(default)]
+    pub stdout: bool,
+    #[serde(default)]
+    pub stderr: bool,
+    #[serde(default)]
+    pub foreground: bool,
```

Comment:
Multiple points here:
1. If the process starts in background, it usually makes sense to drop the stdout and stderr. If the process starts in foreground, it makes more sense to keep the stdout and stderr inherited by default.

    This is a case where using `Option<bool>` gives you an extra "default" value that is useful. If the option is None and the process is foreground you can use `true` by default. While if the option is None and the process is background you can use `false` by default.

2. Here background processes are the default. Should they be the default? Should it be foreground the default? It's actually pretty hard to tell. Probably there shouldn't be a default here and the field should be mandatory.